openldap/assets/dashboards/openldap_overview.json

```diff
@@ -0,0 +1,584 @@
+                    "line_type": "solid",
+                    "line_width": "normal"
+                  },
+                  "display_type": "line"
+                }
+              ],
+              "yaxis": {
+                "scale": "linear",
+                "label": "",
+                "include_zero": true,
+                "min": "auto",
+                "max": "auto"
+              },
+              "markers": []
+            },
+            "layout": {
+              "x": 8,
+              "y": 0,
+              "width": 4,
+              "height": 3
+            }
+          }
+        ]
+      }
+    },
+    {
+      "id": 5418366396402,
+      "definition": {
+        "title": "Operations Metrics",
+        "type": "group",
+        "show_title": true,
+        "layout_type": "ordered",
+        "widgets": [
+          {
+            "id": 2205784706649616,
+            "definition": {
+              "title": "Completed Operations",
+              "title_size": "16",
+              "title_align": "left",
+              "show_legend": true,
+              "legend_layout": "auto",
+              "legend_columns": [
+                "avg",
+                "min",
+                "max",
+                "value",
+                "sum"
+              ],
+              "type": "timeseries",
+              "requests": [
+                {
+                  "q": "avg:openldap.operations.completed.total{$host,$url} by {host,url}.as_count()",
+                  "on_right_yaxis": false,
+                  "style": {
+                    "palette": "dog_classic",
+                    "line_type": "solid",
+                    "line_width": "normal"
+                  },
+                  "display_type": "bars"
+                }
+              ],
+              "yaxis": {
+                "scale": "linear",
+                "label": "",
+                "include_zero": true,
+                "min": "auto",
+                "max": "auto"
+              },
+              "markers": []
+            },
+            "layout": {
+              "x": 0,
+              "y": 0,
+              "width": 6,
+              "height": 4
+            }
+          },
+          {
+            "id": 7619695970441178,
+            "definition": {
+              "title": "Incomplete Operations",
+              "title_size": "16",
+              "title_align": "left",
+              "show_legend": true,
+              "legend_layout": "auto",
+              "legend_columns": [
+                "avg",
+                "min",
+                "max",
+                "value",
+                "sum"
+              ],
+              "type": "timeseries",
+              "requests": [
+                {
+                  "q": "top(avg:openldap.operations.completed.total{$host,$url} by {host,url}.as_count(), 10, 'sum', 'desc')-top(avg:openldap.operations.initiated.total{$host,$url} by {host,url}.as_count(), 10, 'sum', 'desc')",
```

Comment:
Wouldn't incomplete  be `total.initiated - total.completed` ? Also, this is ambiguous because some connections may be inflight and others may have failed.

Reply:
Good point-- would you recommend dropping this completely or more along the lines of renaming or adding a note for clarity?

Reply:
I'd recommend you can add a note for clarity. It is always good to show something that can help identify failures/errors. And it seems like most of the metrics for openLDAP are not error related.

Reply:
Alright! I think this has been updated and has also been pushed to staging: https://dd.datad0g.com/dash/integration/30641/openldap---overview?from_ts=1620379678109&live=true&to_ts=1620394078109